pom.xml

```diff
@@ -18,11 +18,13 @@
         <!-- Pipeline properties -->
         <docker.image.name>ms-email-reader</docker.image.name>
         <exposed.app.port>8080</exposed.app.port>
-        <published.container.port.sandbox>8707</published.container.port.sandbox>
         <published.container.port.test>8706</published.container.port.test>
         <published.container.port.production>8706</published.container.port.production>
-
-        <powermock.version>2.0.2</powermock.version>
+        <!-- Generated sources properties -->
+        <openapi-generator-maven-plugin>6.6.0</openapi-generator-maven-plugin>
```

Comment:
If I'm not mistaken there's a newer 7.0.0 (?) of this plugin

Reply:
You are correct.